examples/diff-revisions.py

```diff
@@ -3,17 +3,21 @@
 import os.path
 
 from ws.client import API
+from ws.client.connection import Connection
 from ws.diff import RevisionDiffer
 
 api_url = "https://wiki.archlinux.org/api.php"
+index_url = "https://wiki.arclinux.org/index.php"
 cookie_path = os.path.expanduser("~/.cache/ArchWiki.cookie")
+session = Connection.make_session(ssl_verify=True,
```

Comment:
You can use `API.make_session` because `API` is a subclass of `Connection`.

Reply:
Done.